__tests__/plugin.test.ts

```diff
@@ -225,7 +225,7 @@ describe('Auth0Plugin', () => {
     expect(handleRedirectCallbackMock).not.toHaveBeenCalled();
 
     return flushPromises().then(() => {
-      expect(replaceStateMock).toHaveBeenCalled();
+      expect(replaceStateMock).not.toHaveBeenCalled();
```

Comment:
This test and implementation was actually incorrect and it should not call replaceState when not handling the redirectCallback.